node/querystring.ts

```diff
@@ -2,7 +2,114 @@
 import { Buffer } from "./buffer.ts";
 import { ERR_INVALID_URI } from "./_errors.ts";
 
-export interface ParsedUrlQuery {
+/******************** querystring.decode() ********************/
+
+/**
+ * Alias of querystring.parse()
+ * @deprecated
```

Comment:
querystring API are tagged as `Legacy` and it seems that `legacy` is different from `deprecated` in Node.js documentation (ref: https://github.com/nodejs/node/pull/37784 for example )

So maybe we should tag this as `@legacy`?

Reply:
:+1: https://github.com/denoland/deno_std/pull/1531/commits/349b5f10310055c3c205b7f11fb2d560efb6d5a9